### PR TITLE
add csv option to uvvis measurement

### PR DESCRIPTION
**Pull request by carla-terboven**

### CODE REVIEW

**Comment by carla-terboven** [comment]
I have used already existing baseclasses. But I am not sure if they are entirely correct because the plot in UVVisData has a strange unit on the x-axis. The wavelength is read in nm, but the plot shows values that are 10 times as large and the unit is displayed as Å.

**Comment by carla-terboven** [comment]
Ok I found that Å is an actual unit and that it can be converted to the correct values in nm. So the problem has resolved itself.

File: ce_nome_s/__init__.py [hunk 1]
```diff
@@ -851,13 +851,18 @@ def normalize(self, archive, logger):
...
                 file_name = f.name
             datetime_object = None
-            if os.path.splitext(data_file)[-1] != ".ABS":
-                continue
-            data = pd.read_csv(
-                file_name, delimiter='  ', header=None, skiprows=2)
-            from baseclasses.helper.archive_builder.uvvis_archive import get_uvvis_archive
-            measurements.append(get_uvvis_archive(
-                data, datetime_object, data_file))
+            if os.path.splitext(data_file)[-1] == ".csv":
```
Comment (RoteKekse) [inline ce_nome_s/__init__.py:854]: i would then suggest: 
```python
if os.path.splitext(data_file)[-1] not in  [".ABS", ".csv"]:
    continue
```

File: ce_nome_s/__init__.py [hunk 2]
```diff
@@ -851,13 +851,18 @@ def normalize(self, archive, logger):
...
+                data = pd.read_csv(
```
Comment (RoteKekse) [inline ce_nome_s/__init__.py:855]: is there any difference in .abs and .csv, if not we can use the same code for both :D

**Review by RoteKekse — CHANGES_REQUESTED** [review]

**Comment by RoteKekse** [comment]
@carla-terboven please add a plot alread to the overview, right now it is only in the subsection

**Comment by carla-terboven** [comment]
The delimiter in the pd.read_csv is different thats why I decided to do it like this.

**Comment by carla-terboven** [comment]
I don't know how to add the plot because there can be multiple measurements. Right know I added a plot with axis but it is not showing anything because I need something like `measurements[0]/intensity`. But this is not working.

**Comment by carla-terboven** [comment]
Ok I found the solution for the plot.

**Review by RoteKekse — APPROVED** [review]

**Comment by carla-terboven** [comment]
> ah ok, i looked very hard but didnt see it :D , then please set the delimiter
> 
> ```python
> delimiter = "," if os.path.splitext(data_file)[-1] ".csv" else "  "
> ```
> 
> to repeat as little code as possible.

Oh I see. That would have been even nicer.

**Pull request merged**